test_exercise7.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+import unittest
+from exercise7 import calculate_room_measurement_feet, convert_feet_to_meters
+
+class RoomCalculationsTest(unittest.TestCase):
+    """Calculations to get area of room"""
+
+
+    def test_room_measure_feet(self):
```

Comment:
The function you're actually testing is `calculate_room_measurement_feet`, so this test should be named `test_calculate_room_measurement_feet`.